_data/yailke123/yailke123.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Yagiz Efe Mertol
+institution: Bilkent University
+profile_pic: yailke123.jpg
```

Comment:
Your profile_pic entry and uploaded image name do not match

Reply:
I believe I managed to fix it. Thanks :)